### PR TITLE
OpenGL: Support blend equation

### LINKED ISSUE #1452: Rune Factory 4 - no colour gate
Another Rune Factory 4 bug, though not as annoying as the corrupted textures.All monster gates appear as completely white vortexes.Depending on the type of gate,a gate supposed to have a specific colour,black on it outer edge, the element or type colour in the mid range, and white at center.
screenshot 38
  log:
https://justpaste.it/ryq1
this is not fixed by #1436

### PR DESCRIPTION
**Pull request by JayFoxRox**
This was missing somehow.
Another one of those "how did that even work"-moments. (Though I must admit, seeing anything but FUNC_ADD is unlikely..)

Fixes #1452  (gates are coloured but I didn't check if it's the correct color, I just assume so)

//Edit: SW renderer already implemented these.

### CODE REVIEW

File: src/video_core/renderer_opengl/gl_state.cpp [hunk 1]
```diff
@@ -165,6 +167,11 @@ void OpenGLState::Apply() const {
                             blend.src_a_func, blend.dst_a_func);
     }
 
+    if (blend.rgb_equation != cur_state.blend.rgb_equation ||
+            blend.a_equation != cur_state.blend.a_equation) {
```
Comment (lioncash) [inline src/video_core/renderer_opengl/gl_state.cpp]: Over-indentation by about 4 spaces.

Comment (JayFoxRox) [inline src/video_core/renderer_opengl/gl_state.cpp]: same as the others in this function (I didn't like it either :wink:)

**Comment by emmauss** [comment]
i will merge and test.

**Comment by Subv** [comment]
LGTM

**Comment by bunnei** [comment]
LGTM

**Pull request merged**